### PR TITLE
product template

### PR DESCRIPTION
**Pull request by Airyoldi**

### CODE REVIEW

**Review by Ludovicscelles — APPROVED** [review]

File: client/src/components/swapProposition/swapProposition.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
+.proposition {
```
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css]: English name class

File: client/src/components/swapProposition/swapProposition.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+h3 {
+  font-family: var(--font);
+  font-size: small;
+}
+.content {
+  display: flex;
+  flex-direction: column-reverse;
+}
+.PropositionProducts {
```
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css]: rename camel case and proposition english

File: client/src/components/swapProposition/swapProposition.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  background-color: rgba(41, 41, 41, 0.5);
+  backdrop-filter: blur(5px);
+}
+
+body.active {
+  overflow: hidden;
+}
+
+@media only screen and (min-width: 769px) {
+  .proposition {
```
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css]: rename in english

File: client/src/components/swapProposition/swapProposition.css [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+    padding: 20px 0px 0px 35px;
+    position: fixed;
+    z-index: 1001;
+    background-color: var(--grey);
+    border-radius: 25px;
+    top: 70px;
+    left: 25%;
+  }
+
+  .PropositionProducts {
```
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css]: rename camel case and proposition english

File: client/src/components/swapProposition/swapProposition.css [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    font-weight: bold;
+    font-size: medium;
+    bottom: 40px;
+  }
+
+  .btnSend:hover {
+    background-color: var(--hover-blue);
+  }
+
+  .PropositionProducts {
```
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css]: rename camel case and proposition english

File: client/src/components/swapProposition/swapProposition.css
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.css:1]: Re check all class name please :)

File: client/src/components/swapProposition/swapProposition.jsx
Comment (Alexandre78R) [inline client/src/components/swapProposition/swapProposition.jsx:1]: Rename folder and file SwapPropisition

File: client/src/pages/Product/ProductPage.css
Comment (Alexandre78R) [inline client/src/pages/Product/ProductPage.css:1]: Rename file productPage

File: client/src/pages/Product/ProductPage.jsx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  };
+
+  useEffect(() => {
+    fetch("http://localhost:3310/api/items/7/user")
+      .then((response) => response.json())
+      .then((json) => setProduct(json[0]));
+  }, []);
+
+  // eslint-disable-next-line no-console
+  console.log(product);
```
Comment (Alexandre78R) [inline client/src/pages/Product/ProductPage.jsx]: delete console. log and delete rules // eslint-disable-next-line no-console

File: client/src/pages/Product/ProductPage.css [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+.swapButton:hover {
+  background-color: var(--hover-blue);
+}
+
+.swapBtn {
+  display: none;
+}
+
+h4 {
```
Comment (Alexandre78R) [inline client/src/pages/Product/ProductPage.css]: Add className

File: client/src/pages/Product/ProductPage.css [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+  padding: 20px 20px 20px 0px;
+  font-family: var(--font);
+}
+
+h2 {
```
Comment (Alexandre78R) [inline client/src/pages/Product/ProductPage.css]: Add className

File: client/src/pages/Product/ProductPage.css [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+    width: 70%;
+    padding: 0px;
+    margin: 20px 0px 0px 40px;
+  }
+
+  .productPage {
+    margin-bottom: 40px;
+  }
+
+  h4 {
```
Comment (Alexandre78R) [inline client/src/pages/Product/ProductPage.css]: Add className

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

File: client/src/pages/Product/productPage.css [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+    width: 70%;
+    padding: 0px;
+    margin: 20px 0px 0px 40px;
+  }
+
+  .productPage {
+    margin-bottom: 40px;
+  }
+
+  h4 {
```
Comment (Alexandre78R) [inline client/src/pages/Product/productPage.css]: replace name class please

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

File: client/src/components/SwapProposition/swapProposition.css [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+}
+
+.top {
+  display: flex;
+  align-items: center;
+  position: relative;
+  right: 10px;
+}
+
+h3 {
```
Comment (Alexandre78R) [inline client/src/components/SwapProposition/swapProposition.css]: add classname

File: client/src/components/SwapProposition/swapProposition.css [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+    top: 20px;
+  }
+  .top {
+    display: flex;
+    align-items: center;
+    position: relative;
+    left: 10px;
+  }
+
+  h3 {
```
Comment (Alexandre78R) [inline client/src/components/SwapProposition/swapProposition.css]: add classname

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by Alexandre78R — APPROVED** [review]

**Pull request merged**